stock_warehouse_calendar/tests/test_stock_warehouse_calendar.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2018-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestStockWarehouseCalendar(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
FYI, you can use https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L21

Reply:
Hi @rousseldenis , thank you very much for the review. Is it better like this?

https://github.com/OCA/stock-logistics-warehouse/compare/e211862a6943610f2a678ffa44dffc01eb4362e1..efcfd5befe89f7f39209591d8675f98f5f77cd2c``

Reply:
@RicardCForgeFlow You can remove `TransactionCase` :sweat_smile:

Reply:
Oups, that's true 😳